### DISCUSSION TITLE
Simple Question

### DISCUSSION THREAD

**Discussion by ObviouslyAxo** [FabricMC/fabric-api#3577] [FAQ] [answered]
I want to start using mods for Minecraft and my friend said I might need to download java but I do not know. Sorry if I wasted your time

**Comment by modmuss50** [comment, marked as answer]
Assuming you are using the vanilla (the official) Minecraft launcher on Windows then its unlikely you will need to manually download Java.

This is becuase the official launcher downloads Java for you alongside Minecraft, the Fabric installer for Windows is able to find this version of Java, removing the need for you to download it.

**Reply by ObviouslyAxo** [reply]
Thank You For The Help!
